### ISSUE TITLE
Missing XML sources.

### ISSUE THREAD

**Issue by iBotPeaches** [MiCode/patchrom_miui#4]
I'm building  English Translations into the XML you provided using xml tools, but yet when converting.

I don't see

framework-res
framework-miui-res
downloadProviderUi

And thats all I remember. Were those XML sources not included?

**Comment by Benson17** [comment]
Push - any news on this, why it is missing? Can we help to gather the needed files? At least we need framework-res and framework-miui-res, so we could build a translation pack with git as source. Thanks!

**Comment by iBotPeaches** [comment]
I manually found them, but it doesn't work with our current Apktool for
rebuild. Apktool hasn't been updated in 4 months and it doesn't like the
xlif data and what not. So it can't rebuild using raw xml.

However its a good start. I tried fixing Apktool, but it took me weeks to
fix one bug. See my repos for the work in progress.

Sent from an HTC Inspire 4G running MIUI.
On Apr 17, 2012 3:39 AM, "Benson17" <
reply@reply.github.com>
wrote:
